<launch>
    <node name = "ik_service" pkg = "ik_service" type = "ik_service_node"/>
</launch>
